tests/aws/services/lambda_/test_lambda_api.py

```diff
@@ -6621,6 +6623,33 @@ def test_layer_policy_lifecycle(
             "get_layer_version_policy_postdeletes2", get_layer_version_policy_postdeletes2
         )
 
+    @markers.aws.only_localstack(reason="Deterministic id generation is LS only")
```

Comment:
A quick explanation why this is needed might be helpful. Our future selfs thank us :)